File: api/src/main/java/com/example/api/dto/enums/BookingState.java
package com.example.api.dto.enums;

/**
 * Критерий поиска запросов на бронирование.
 */
public enum BookingState {

    /**
     * Все.
     */
    ALL,

    /**
     * Текущие.
     */
    CURRENT,

    /**
     * Будущие.
     */
    FUTURE,

    /**
     * Завершённые.
     */
    PAST,

    /**
     * Отклонённые.
     */
    REJECTED,

    /**
     * Ожидающие подтверждения.
     */
    WAITING
}

File: api/src/main/kotlin/com/example/api/dto/enums/BookingState.kt
package com.example.api.dto.enums

/**
 * Критерий поиска запросов на бронирование.
 */
enum class BookingState {
    
    /**
     * Все.
     */
    ALL,

    /**
     * Текущие.
     */
    CURRENT,

    /**
     * Будущие.
     */
    FUTURE,

    /**
     * Завершённые.
     */
    PAST,

    /**
     * Отклонённые.
     */
    REJECTED,

    /**
     * Ожидающие подтверждения.
     */
    WAITING
}
